docs/manage/fleet/provision/end-user-setup.md

```diff
@@ -135,6 +138,59 @@ Note that any features that require internet access will not function if the con
 {{% /tablestep %}}
 {{< /table >}}
 
+## Set up your machine using a custom Flutter app
+
+If you are building your own app to provide provisioning functionality you have three options for provisioning:
```

Comment:
@kevin49999 do you mind taking another look here:

Is this table covering this correctly?

Table is easier to look at here: https://deploy-preview-4372--viam-docs.netlify.app/manage/fleet/provision/end-user-setup/#set-up-your-machine-using-a-custom-flutter-app